website/templates/public/pages/meeting_body.mako

```diff
@@ -7,21 +9,26 @@
 
 % if meeting['active']:
     <div>
-        <a id="addLink" onclick="" href="#">Add your poster or talk</a>
+        <a id="addLink" onclick="" href="#">${'Add your posters or talks' if meeting['poster'] and meeting['talk'] else 'Add your posters' if meeting['poster'] else 'Add your talks'}</a>
+
         % if meeting['info_url']:
           | <a href="${ meeting['info_url'] }" target="_blank">Conference homepage <i class="fa fa-sm fa fa-external-link"></i></a>
         % endif
     </div>
 
     <div style="display: none" id="submit">
-        <h3>Add your poster or talk</h3>
+        <h3>${'Add your posters or talks' if meeting['poster'] and meeting['talk'] else 'Add your posters' if meeting['poster'] else 'Add your talks'}</h3>
         <p>
             Send an email to one of the following addresses from the email
```

Comment:
It says "one of the following addresses", even for conferences where there is now only one item in the list.